bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcPanel.scala

```diff
@@ -47,6 +47,12 @@ sealed trait ItcPanel extends GridBagPanel {
   def display: Component
   def visibleFor(t: SPComponentType): Boolean
 
+  /** TODO: we need to display a list of asterism members and let the user select one. For now we
+    * just select the first target in the asterism.
+    */
+  def selectedTarget: Option[Target] =
+    Option(owner.getContextTargetEnv).map(_.getAsterism.allTargets.head)
+
```

Comment:
The `ItcPanel` needs to add a new UI element to let you select the asterism component. Right now it just uses the first target. This was easy to stub out but other UI elements won't be so easy.